mimic/model/nova_objects.py

```diff
@@ -527,6 +527,35 @@ def fail_later(server):
     return fail_later
 
 
+@server_creation.declare_behavior_creator("sequence")
+def sequence(parameters):
+    """
+    Sometimes a sequence of behaviors occur when you try to create a server in
+    a predictable pattern.
+
+    Takes one parameter, ``behaviors``, which is a list of specifications of
+    other behaviors.
+    """
+    behavior_specification = parameters["behaviors"]
+    behavior_objects = [
+        (
+            server_creation.create_behavior(behavior["name"],
+                                            behavior["parameters"])
+            if behavior["name"] != "default"
+            else server_creation.default_behavior
+        )
+        for behavior in behavior_specification
+    ]
+
+    def rotating_behavior(collection, http, json, absolutize_url):
+        current = behavior_objects[rotating_behavior.index]
```

Comment:
I think you mean `itertools.cycle`?  I explicitly wanted to do repeating in a separate pull request, though, as the description said.